### PR TITLE
feat: Configure DW site default navigation - EXO-65024_EXO-65400 - Meeds-io/MIPs#72

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties [hunk 1]
```diff
@@ -19,3 +19,6 @@
 portal.dw.home=Snapshot
```
Comment (azayati) [inline digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties:19]: should we remove this label ?

File: digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties [hunk 2]
```diff
@@ -19,3 +19,6 @@
...
 portal.dw.externalHome=Home
 portal.dw.overview=Participate
```
Comment (azayati) [inline digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties:21]: should we remove this label ?

**Review by azayati — CHANGES_REQUESTED** [review]

File: digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties (hunk 1 above)
Comment (GouadriaHanen) [reply digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties:19]: No it shouldn't for the existing data case

File: digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties (hunk 2 above)
Comment (GouadriaHanen) [reply digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties:21]: same here

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**